docs/Documentation/Environment/shell.md

```diff
@@ -1,68 +1,71 @@
+*When you login to a linux-based machine you interact with the operating system via a program called a shell.  There are various types of shell programs.  One of the more common is **bash**.  Bash is the default shell on NREL's HPC platforms.  This document describes ways you can customize your shell's &#151; in particular, bash's &#151; behavior.*
 
 
 
 ## Getting Started
-When you have a window open attached to a platform you are actually running a program on the remote computer, called a shell.  There are various types of shell programs.  One of the more common is *bash*.  
+When you have a window open attached to a platform, you are actually running a program on the remote computer called a shell.  There are various types of shell programs.  One of the more common is *bash*.  
 
-The shell program provides your link to the machine's operating system (OS).  It is the interface between a user and the computer. It controls the computer and provides output to the user.  There are various types of interfaces but here we discuss the command line interface. That is, you type commands and the computer responds.
+The shell program provides your link to the machine's operating system (OS).  It is the interface between a user and the computer. It controls the computer and provides output to the user.  There are various types of interfaces but here we discuss the command-line interface. That is, you type commands and the computer responds.
 
 ### What happens on login
 
-When you login to a machine you are put in your home directory.  You can see this by running the command **pwd**.  Run the command **ls -a** to get a listing of the files.  The **-a** option for the ls commands enables it to show files that are normally hidden.  You'll see two important files that are used for setting up your environment.
+When you login to a machine, you are put in your home directory.  You can see this by running the command **pwd**.  Run the command **ls -a** to get a listing of the files.  The **-a** option for the ls commands enables it to show files that are normally hidden.  You'll see two important files that are used for setting up your environment.
 
 * .bash_profile
 * .bashrc
 
 
 These files are added to your home directory when your account is created.  
 
-When you login the file .bash_profile is sourced (run) to set up your environment.  The environment includes settings for important variables, command aliases, and functions.  
+When you login, the file .bash_profile is sourced (run) to set up your environment.  The environment includes settings for important variables, command aliases, and functions.  
 
-Here is the default version of .bash_profile.  
+Here is the default version of .bash_profile:  
 
 ```
-[nreluser@el3 ~]$ cat ~/.bash_profile
+[nreluser@kl2 ~] CPU $ cat /etc/skel/.bash_profile
```

Comment:
I think it makes sense to leave the path as it was, so the user knows where the paths to their .bash_profile and .bashrc files are.